requirements.txt

```diff
@@ -1,10 +1,10 @@
 caspo
-joblib==0.10.0
-networkx==1.11
-numpy==1.11.1
-pandas==0.18.1
-pydotplus==2.0.2
-pyparsing==2.1.1
-scikit-learn==0.17.1
-scipy==0.18.0
-seaborn==0.7.0
+joblib
+networkx
+numpy
+pandas
+pydot
+pyparsing
+scikit-learn
+scipy
+seaborn
```

Comment:
I'd pin specific (latest if possible) versions in both requirements.txt and meta.yaml before making the next release

Reply:
Ha, I'm quite against this :wink:  I think it creates much more troubles than it solves whenever other packages are installed alongside. I prefer having early bug reports due to API breakage, and (validated) Docker images really help on this; I also plan to include caspo in the CoLoMoTo Docker ecosystem https://colomoto.github.io/colomoto-docker where we work for providing reproducible software environments.